### PR TITLE
OSDOCS#7865: Migrating Azure DevOps with Managed OCP from MOBB to ROSA

### PR DESCRIPTION
**Pull request by skopacz1**
Versions: 4.13+

Issue: [OSDOCS-7865](https://issues.redhat.com/browse/OSDOCS-7865)

QE review:
- [ ] QE has approved this change.

Preview: [Using Azure DevOps with Managed OpenShift](https://65072--docspreview.netlify.app/openshift-rosa/latest/rosa_tutorials/rosa-mobb-azure-devops-managed-ocp)

### CODE REVIEW

**Comment by skopacz1** [comment]
@michaelryanmcneill Could you PTAL? Thanks

File: rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+$ git clone https://github.com/rh-mobb/azure-pipelines-openshift
+
+$ cd azure-pipelines-openshift
+----
+
+== Configuring Azure Devops
+
+. While logged into your link:https://aex.dev.azure.com/me?mkt=en-US[Azure DevOps Organization], create a new project.
++
+The examples in this document will assume the project is named `azure-pipelines-openshift`
```
Comment (jneczypor) [inline rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc:46]: Consider adding ending punctuation.

File: rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+$ AZP_POOL=Default
+----
+
+== Host an Azure Pipelines Build Agent in OpenShift
+[NOTE]
+====
+This is an abbreviated version of this link:https://cloud.redhat.com/blog/hosting-an-azure-pipelines-build-agent-in-openshift[blog] by Kevin Chung and Mark Dunnett.
+====
+
+In this step, we will configure OpenShift to build our container image leveraging Universal Base Images (UBI)
```
Comment (jneczypor) [inline rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc:80]: Add end punctuation.

File: rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+image::docker-registry.png[Docker Registry option]
+
+. Enter the settings we retrieved in the previous step:
+** Select Others for Docker Registry.
+** Docker Registry - make sure to add `https://` in front of the hostname you retrieved.
+** Docker ID - the service account you created.
+** Docker Password - the service account token.
+** Service connection name - enter `openshift-registry`.
+
+. Next, let's create another service connection for our cluster.
```
Comment (jneczypor) [inline rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc:293]: The contractions could hinder translation. You could just remove the "let's".

File: rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc [hunk 4]
```diff
@@ -0,0 +1,365 @@
...
+. Obtain a personal access token.
+While in Azure DevOPs, select Personal Access Token under User Settings.
++
+image::azpat.png[Azure Personal access tokens]
+
+. On the next screen, create a New Token.
++
+In this example, we will create a token with Full Access.
+Once you click *Create* your token will be displayed.
+Make sure to store it somewhere safe as you won't be able to see it again.
```
Comment (jneczypor) [inline rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc:59]:
```suggestion
Make sure to store it somewhere safe as you will not be able to see it again.
```

**Review by jneczypor — COMMENTED** [review]
Just a couple of small things. LGTM!

File: rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc [hunk 5]
```diff
@@ -0,0 +1,365 @@
...
+
+// ---
+// date: '2023-03-14'
+// title: Azure DevOps with Managed OpenShift
+// tags: ["DevOps", "ADO", "Azure", "ARO", "ROSA"]
+// authors:
+//   - Kevin Collins
+// ---
+
+Adopted from link:https://cloud.redhat.com/blog/hosting-an-azure-pipelines-build-agent-in-openshift[Hosting an Azure Pipelines Build Agent in OpenShift]
```
Comment (laubai) [inline rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc:17]: Adapted?

Comment (skopacz1) [reply rosa_tutorials/rosa-mobb-azure-devops-managed-ocp.adoc:17]: Good catch, I'm wondering if that line should even be there in the first place haha, but in the meantime I'll make this fix

**Comment by skopacz1** [comment]
Closing this for now since the JIRA issue has been moved to the backlog.

**Pull request closed without merging**